src/setupDAL.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+export const DAL = () => {
```

Comment:
setupDAL(). it's a function and the name should start from a verb for better readability